RELEASE_INFO-6.7.md

```diff
@@ -53,6 +53,11 @@ If the token has already been consumed, the user will be redirected directly to
 To support this behavior, a new `consumed` flag has been added to the payment token struct, which indicates if the token has already been processed.
 Payment tokens are no longer deleted immediately after use. A new scheduled task automatically removes expired tokens to keep the `payment_token` table clean.
 
+### Primary key validation in `dal:validate` command
+The `dal:validate` command now includes validation to detect mismatches between database PRIMARY KEY constraints and entity definition PrimaryKey flags.
```

Comment:
nice, that the validation is now supporting this, but this will not prevent the wrong API response. this should imo also be tackled

Reply:
just throw a proper exception like we do with other SQL errors :man_shrugging: